src/linear_solver/NNLS_solver.h

```diff
@@ -131,10 +133,11 @@ class NNLS_solver
     const dealii::ParameterHandler &parameter_handler;
 
 protected:
-    const Epetra_CrsMatrix A_;
     Epetra_MpiComm Comm_;
+    const Epetra_CrsMatrix A_;
```

Comment:
Could you make sure that the documentation comments in this class comply with the Doxygen conventions of `/// comment` on the line before or `///<` immediately following a variable?

Reply:
Completed in latest commit for all variables in NNLS_solver.h

Reply:
Great, thanks!